packages/p2p/src/components/orders/order-details/order-details.jsx

```diff
@@ -214,8 +266,23 @@ const OrderDetails = ({
             </div>
 
             <FooterActions>
-                <OrderActionsBlock order_details={ order_details } />
+                <OrderActionsBlock cancelPopup={onCancelClick} showPopup={handleShowPopup} order_details={ order_details } />
             </FooterActions>
+            {show_popup && (
+                <div className='orders__dialog'>
+                    <Dialog
+                        is_visible={show_popup}
+                        disableApp={() => {
+                            /* do nothing // disableApp is a mandatory props in dialog */
+                        }}
+                        enableApp={() => {
+                            /* do nothing // enableApp is a mandatory props in dialog */
```

Comment:
should we make it not mandatory instead since it doesn't need to be used all the time?